cmd/main.go

```diff
@@ -165,6 +165,11 @@ func NewMainApp(appVer AppVersion) *cli.App {
 	app.Commands = append(app.Commands, subCmdWithConfig...)
 	app.Commands = append(app.Commands, subCmdStandalone...)
 
+	err := os.Unsetenv("XDG_CONFIG_HOME") // unset if set as HOME is managed by gitea
+	if err != nil {
+		fmt.Printf("error unsetting XDG_CONFIG_HOME")
+	}
+
```

Comment:
```suggestion
	_ = os.Unsetenv("XDG_CONFIG_HOME") // unset if set as HOME is managed by gitea
```

It can't (not recommend to) use "Printf" because IIRC there is a lint rule 🤣